xscen/data/file_schema.yml

```diff
@@ -26,46 +24,65 @@
 # It includes bias-adjusted datasets.
 raw-non-sims:
   with:
-    - option: type
+    - facet: type
       value: [station-obs, reconstruction, forecast]
-    - option: processing_level
+    - facet: processing_level
       value: raw
   folders:
     - type
     - domain
     - institution
     - [source, version]
-    - member
+    - (member)
     - frequency
     - variable
   filename: [ variable, frequency, domain, institution, source, version, member, DATES ]
-raw-sims:
+raw-sims-raw:
```

Comment:
we use raw in two different contexts:
1) raw vs bias adjusted
2) raw vs derived

Can we find a different word for 2 ? I think it would be less confusing and avoid "raw-sims-raw".
a few suggestions: `direct`, `basic`, `original`,  `primary`, `elemental` ?

Reply:
I prefer `original` from this list.